.github/workflows/integration_tests.yml

```diff
@@ -22,9 +22,19 @@ defaults:
   run:
     shell: bash
 jobs:
+  calculate-queue-time:
+    name: "Calculate Queue Time"
+    uses: keptn/keptn/.github/workflows/pipeline-queuetime.yml@master
```

Comment:
I recommend properly re-use workflows from the current branch, e.g.:
```
     uses: ./keptn/keptn/.github/workflows/pipeline-queuetime.yml
```
(Note the `./` in the beginning and the missing branch name at the end)

Reply:
aha! i didn't know that that's possible. so when not defining the branch it will take the current one?